uber/templates/preregistration/preregbase.html

```diff
@@ -239,6 +238,7 @@
                         $.field('amount_extra').on('change', makeBadgeMatchExtra);
                     {% endif %}
                 }
+                togglePrices();
```

Comment:
The diff is annoying but this is the core change that fixes the bug. We now trigger togglePrices() after printing all the badge_types stuff on the page. The only reason we ever saw prices on the reg form was because every click event on a reg_types button also triggers togglePrices, and we automatically trigger a click event on the Attendee reg type button when we load the prereg form.